SparkPost/2web2ui
# AC-1595 - Remove the GET /billing/subscription call for Heroku and Azure users

AC-1595 - Remove the GET /billing/subscription call for Heroku and Azure users

### What Changed
 - Removed the GET /billing/subscription call for Heroku and Azure users from accessControlState
*Note - First commit is the actual change, second one is snapshot updates*
### How To Test
 - To login as a heroku user, use the link in chat https://msys.slack.com/archives/C69CBU1HP/p1596055498030400?thread_ts=1595971910.001200&cid=C69CBU1HP
- Verify that app loads.

### To Do
